client/app/async-loader.factory.js

```diff
@@ -0,0 +1,46 @@
+// Inspired by angular-translate-partial-loader (see sources)
```

Comment:
Could you provide an URL related to `see sources`?

Reply:
This loader should be moved, ping @frenautvh :)